package.json

```diff
@@ -23,22 +23,24 @@
   "module": "dist/vue-imgix.esm.js",
   "jsnext:main": "dist/vue-imgix.esm.js",
   "unpkg": "dist/vue-imgix.min.js",
-  "version": "2.8.4",
+  "version": "3.0.0-rc.4",
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "rollup --config build/rollup.config.js",
     "build:test-app": "vue-cli-service build",
     "test": "run-s test:unit test:e2e",
     "test:unit": "vue-cli-service test:unit",
-    "test:e2e": "vue-cli-service test:e2e",
+    "test:e2e": "vue-cli-service test:e2e --browser chrome",
     "lint": "vue-cli-service lint",
     "lint:ci": "vue-cli-service lint --no-fix --max-warnings 0",
-    "prepublishOnly": "run-s build"
+    "prepublishOnly": "run-s build",
+    "release:dryRun": "npx node-env-run --exec 'npx semantic-release --dryRun'"
   },
   "dependencies": {
-    "@imgix/js-core": "^3.1.3"
+    "@imgix/js-core": "^3.6.0-rc.1"
```

Comment:
We might want to bump this to `3.6.0` proper soon since it's been released. Follow up PR after `3.0` release maybe.